ci/test_wheel_cudf_polars.sh

```
@@ -51,14 +33,41 @@ set +e
+    polars_requirements_txt="polars-compat-${version}-requirements.txt"
+    rapids-dependency-file-generator \
+        --config dependencies.yaml \
+        --file-key test_cudf_polars_compat \
+        --output requirements \
+        --matrix "polars_compat_version=${version}" \
+        > "${polars_requirements_txt}"
+
+    env_name="venv_polars_${version}"
+    python -m venv --clear "${env_name}"
+    # shellcheck disable=SC1090
+    source "${env_name}/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./bin/activate was not specified as input (see shellcheck -x).

(SC1091)